saleor/graphql/order/types.py

```diff
@@ -1,11 +1,41 @@
 import graphene
 from graphene import relay
-from graphene_django import DjangoObjectType
-
-from ...order import models
+from graphene.types import Scalar
+from ...order import OrderEvents, models
 from ..core.types.common import CountableDjangoObjectType
 from ..core.types.money import Money, TaxedMoney
 
+OrderEventsEnum = graphene.Enum.from_enum(OrderEvents)
+
+
+class ParametersScalar(Scalar):
+
+    @staticmethod
+    def parse_value(value):
+        return value
+
+    @staticmethod
+    def serialize(dict_parameters):
+        if isinstance(dict_parameters, dict):
+            return dict_parameters
+        return None
+
+    @staticmethod
+    def parse_literal(node):
+        return node
+
+
+class OrderEvent(CountableDjangoObjectType):
+    event = OrderEventsEnum(description='Order event type')
+    parameters = ParametersScalar(
```

Comment:
I think we could do a better job here by just explicitly listing the fields we have along with their types. Otherwise we'll have to handle a custom scalar type in client code and maintain documentation out-of-channel (as the internal structure of a custom scalar cannot be expressed using the schema).